File: solutions/problem_0017/C/Main.c
/**
 * Elaborado por: Ednilson Rossi
 * Data.........: 24/03/2024
 * 
 * Solução iniciante, sem utilização de funções matemáticas. Objetivo de introdução à
 * programmação em linguagem C.
 * 
 * Utiliza-se apenas o tipo inteiro para pratica do raciocínio lógico.
*/

#include <stdio.h>

int main()
{
    int tempo, vel_media, distancia, consumo;

    printf("Tempo da viagem (em horas): \n");
    scanf("%d", &tempo);

    printf("Velocidade média (em km/h): \n");
    scanf("%d", &vel_media);

    distancia = vel_media * tempo;

    consumo = distancia / 12;

    printf("Velocidade média: %d km/h \n", vel_media);
    printf("Tempo de viagem: %d horas \n", tempo);
    printf("Distância percorrida: %d km \n", distancia);
    printf("Consumo de combustível: %d litros \n", consumo);
    
    return 0;
}

File: solutions/problem_0017/Python/main.py
#--- Elaborado por: Janaina Abib    ---#
#--- Data: 24/03/2024               ---#
#--------------------------------------#

def principal( ):
    tempo = float(input("Informe a quantidade de horas que a viagem durou: "))
    velMedia = float(input("Informe a velocidade média durante a viagem: "))

    distancia = (tempo * velMedia)
    litros = (distancia / 12)
    print("A velocidade média foi de {:.2f}".format(velMedia))
    print("O tempo gasto na viagem foi de {:.2f} horas".format(tempo))
    print("A distância total percorrida foi de {:.2f}KM".format(distancia))
    print("Foram gastos durante a viagem {:.2f} litros de combustível".format(litros))
#-----------------------------------FIM---------------------------------
principal( )

